src/System.Private.CoreLib/src/System/Globalization/GlobalizationMode.Unix.cs

```diff
@@ -13,12 +13,17 @@ private static bool GetGlobalizationInvariantMode()
             bool invariantEnabled = false;
             if (!invariantEnabled)
             {
+                // WASM TODO: There's no WASM build of LibICU. We may be able to cross-compile it ourselves.
```

Comment:
I doubt that LibICU would make sense for WASM. It is big. I think we should use the Invariant or Dummy globalization for WASM. These two should be merged eventually somehow - the Dummy globalization was invented before we had the Invariant globalization.

Reply:
Are we able to control that kind of thing in ILC? Since WASM is likely to be used in UIs, some may want localization while others would prefer a smaller size.

Reply:
There is no control for it in ILC yet, but I expect we will want to have one eventually.

Feel free to ignore my comment.

Reply:
You raise a good point. For size conscious folks like WASM users (or even the folks discussing #5564), it would be nice to have control over major size contributors like globalization.